src/redeem.ts

```diff
@@ -143,6 +144,17 @@ export class Redeem {
             console.log("Parachain not connected");
             return;
         }
+        console.log(`[${new Date().toLocaleString()}] -----Cancelling expired redeems-----`);
+        const botAccountId = this.polkaBtc.api.createType("AccountId", account.address);
```

Comment:
I think it would be great to do this type of conversion in the lib instead of in the bot.

Reply:
We should get rid of all the `AccountId` objects from the lib interface, I agree. But should this be done now or a bit later?

Reply:
Can do later